src/components/Chatting.tsx

```diff
@@ -146,8 +153,9 @@ export const Chatting = () => {
           {chatData.chat
             .filter((chat) => chat.c_id !== 0)
             .map((chat, index, arr) => {
-              //만약 그 전 시간과 같다면 그 전에서 시간이 안 보이게하고 마지막에만 보이도록
-              const showTime = index === arr.length - 1 || chat.time !== arr[index + 1].time
+              //만약 그 전 채팅시간과 같다면 그 전 채팅시간이 사라지고 마지막 채팅에만 보이도록
+              const showTime =
+                index === arr.length - 1 || chat.time !== arr[index + 1].time || chat.from !== arr[index + 1].from
```

Comment:
```suggestion
const showTime: boolean =
  index === arr.length - 1 ||
  chat.time !== arr[index + 1].time ||
  chat.from !== arr[index + 1].from;
```
`Typescript`를 활용하여 type을 지정해주면 코드가 더 명확해질 거 같습니다! 혹은 아예 새로운 함수를 선언해서 view와 logic을 분리할 수 있을 것 같아용

Reply:
type지정해주었습니다...!